### ISSUE TITLE
Long term move to PostgreSQL database

### ISSUE THREAD

**Issue by isedwards** [opencdms/Climsoft#488]
From issue #193:

> The roadmap for Climsoft V5 involves changing to PostgreSQL to take advantage of better support for areal data

> PostgreSQL tends to be the standard open source replacement for Oracle DB

There are other reasons for moving to PostgreSQL/PostGIS for the web app release v6

**Comment by mhabimana** [comment]
@isedwards: National Meteorological Services are nowadays interested gridded with gridded data,some of these are the blended data from both point data and sattellite data but the current version of Climsoft is not able to store such kind of data,we need to think of this as well as and make sure we give room to such important type of data.

**Comment by isedwards** [comment]
I agree - this is one of the key arguments for moving to PostgreSQL/PostGIS - the support for gridded data is excellent.  For example, if you a satellite image showing surface temperatures of East Africa and you had a shapefile of the outline of Lake Victoria then you could extra just the temperature over he lake and then use these for further analysis (e.g. calculating the average, or temperature anomalies).

For some details see the PostGIS [Raster reference](https://postgis.net/docs/RT_reference.html)
